Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:alpine AS builder
+
+ARG API_URL
+ENV REACT_APP_BACKEND_URL $API_URL
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY . .
+
+RUN npm install
+
+RUN npm run build
+
+FROM nginx
+
+COPY --from=builder /app/build /usr/share/nginx/html
+COPY nginx.conf /etc/nginx/conf.d/default.conf
```

Comment:
optional: you can make a separate file for nginx

Reply:
but I need to use the build files from Node Frontend...not sure how I can do that with separate files